.pre-commit-config.yaml

```diff
@@ -1,3 +1,5 @@
+fail_fast: True
+
```

Comment:
Would you consider implementing pre-commit.ci? I can handle that part if you want, it just takes some minutes. This would allow to remove the code-style stage, automatically update the pre-commit hooks and a new fancy badge for the main README :)

I should be a salesman 😄

Reply:
Looool

I would do!

Reply:
Thank you for the suggestion :)

Reply:
Awesome! Let me implement it then :)